runtime/src/accounts.rs

```diff
@@ -658,6 +659,14 @@ pub fn create_test_accounts(
     }
 }
 
+pub fn update_accounts(accounts: &Accounts, pubkeys: &[Pubkey], slot: u64) {
+    for pubkey in pubkeys {
+        let amount = thread_rng().gen_range(0, 10);
+        let account = Account::new(amount, 0, &Account::default().owner);
+        accounts.store_slow(slot, &pubkey, &account);
```

Comment:
Is there any specific reason for not using our bulk-version (`accounts::accounts_db.store`) of this? If so, comment is desirable.

But, I think giving a bunch of accounts to the inner `AccountsDB::store_accounts` is more fun and realistic benchmark. :)

Reply:
Yea, this is just for the setup, not really intended to be in the timed part of the benchmark. I actually did experiment with multi-threading and using the accounts_db.store and it didn't make a crazy difference, so I just stuck with a simple one.